tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature

```diff
@@ -644,3 +644,11 @@ Feature: Share by public link
     When the user uploads file "lorem.txt" keeping both new and existing files using the webUI
     Then file "lorem.txt" should be listed on the webUI
     And file "lorem (2).txt" should be listed on the webUI
+
+  Scenario: user creates multiple public links with same name for the same file/folder
```

Comment:
Scenario says the public link for file/folder, but you only test for a folder. Can you please also add test for a file, if possible using Scenario Outline.